### PR TITLE
Feat/manager-7636-volume-backup-removal: removal volume backup modal

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-7636-volume-backup`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-7636
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. modal to remove a volume backup
2. link **Créer un volume** to create block storage volume
3. some improvements

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/controller.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+export default class VolumeBackupDeleteController {
+  /* @ngInject */
+  constructor($translate, VolumeBackupService) {
+    this.$translate = $translate;
+    this.volumeBackupService = VolumeBackupService;
+
+    this.DELETE_CONFIRMATION_INPUT = /^DELETE$/;
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/controller.js:7]: **nitpick:** Already defined in `packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/constants.js`. Re-use

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/template.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <oui-modal
+        data-heading="{{:: 'pci_projects_project_storages_volume_backup_list_delete_title' | translate }}"
+        data-primary-label="{{:: 'pci_projects_project_storages_volume_backup_list_delete_action_confirm' | translate }}"
+        data-secondary-label="{{:: 'pci_projects_project_storages_volume_backup_list_delete_action_cancel' | translate }}"
+        data-primary-disabled="$ctrl.isDeleting || volumeBackupDelete.$invalid"
+        data-secondary-action="$ctrl.onDeleteVolumeBackupCancelClick()"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.isDeleting"
+        data-type="warning"
+    >
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/template.html:16]: **nitpick:** Use `ng-form` and remove the native form element
```suggestion
    <oui-modal
        data-ng-form="volumeBackupDelete"
        data-heading="{{:: 'pci_projects_project_storages_volume_backup_list_delete_title' | translate }}"
        data-primary-label="{{:: 'pci_projects_project_storages_volume_backup_list_delete_action_confirm' | translate }}"
        data-secondary-label="{{:: 'pci_projects_project_storages_volume_backup_list_delete_action_cancel' | translate }}"
        data-primary-disabled="$ctrl.isDeleting || volumeBackupDelete.$invalid"
        data-secondary-action="$ctrl.onDeleteVolumeBackupCancelClick()"
        data-on-dismiss="$ctrl.goBack()"
        data-loading="$ctrl.isDeleting"
        data-type="warning"
    >
```

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/template.html:16]: yes it's more relevant in this case.

File: packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/controller.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/controller.js:7]: forget to remove the constants file.
No more required.

done.

File: packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/template.html (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/template.html]: done

**Review by anooparveti — APPROVED** [review]

**Review by y4nnL — APPROVED** [review]

**Pull request merged**